docs/source/options/preprocess.rst

```diff
@@ -1,7 +1,7 @@
-Preprocess
-==========
+Get vocab
+=========
 
```

Comment:
I switch from `build_vocab` to `get_vocab` because this script gives us the `.txt` format vocab file rather than the `.pt` format we used in training. We still need to do the first phase of training to dump the true `.vocab.pt` file. But if OpenNMT-tf gives it that name, we could use that also.

Reply:
https://opennmt.net/OpenNMT-tf/vocabulary.html
The vocab is also saved in `.txt` format, so keeping the build_vocab name would be consistent I think.